R/repr_matrix_df.r

```diff
@@ -77,19 +95,13 @@ ellip.limit.arr <- function(
 		rv <- cbind(a[, left, drop = FALSE], ellip.h, a[, right, drop = FALSE], deparse.level = 0)
 	}
 
-	if (rows < nrow(a)) {
-		# If there were no rownames before, as is often true for matrices, assign them.
-		if (is.null(rownames(rv)))
-			rownames(rv) <- c(top, ellip.v, bottom)
-		else
-			rownames(rv)[[ top[[length(top) ]] + 1L]] <- ellip.v
+	if (rows < nrow(a) && (! is.null(rownames(rv)))) {
+		# If there were no rownames before, as is often true for matrices, don't assign them.
```

Comment:
the comment is still in place even if you removed the logic.

and why remove the logic at all? did it not work?